index.html

```diff
@@ -204,7 +204,29 @@ <h3>Example</h3>
     "lprNumber": "999-999-999",
     "commuterClassification": "C1",
     "birthCountry": "Bahamas",
-    "birthDate": "1958-07-17"
+    "birthDate": "1958-07-17",
+    "height": [
```

Comment:
This usage doesn't look correct.  It's saying the height is both 5ft and also 11in.  Perhaps it should be only 5.92ft or only 71in?